src/oop/oop2.py

```diff
@@ -17,7 +18,13 @@ def __init__(self, num_wheels):
 #
 # Override the drive() method in Motorcycle so that it returns "BRAAAP!!"
 
-# TODO
+class Motorcycle(GroundVehicle):
+        def __init__(self, num_wheels = 2):
+            self.num_wheels = num_wheels
+            super().__init__(self.num_wheels)
```

Comment:
this does the job, but it'd be a bit cleaner to have something like...

class Motorcycle(GroundVehicle):
    def __init__(self):
        super().__init__(2)